bin/replace

```diff
@@ -9,4 +9,6 @@ shift
 replace_with="$1"
 shift
 
-ag -l --nocolor "$find_this" "$@" | xargs sed -i '' "s/$find_this/$replace_with/g"
+items=$(ag -l --nocolor "$find_this")
+temp="/tmp/replace_temp_file"
```

Comment:
If you want to get fancy:

```sh
temp="${TMPDIR:-/tmp}/replace_temp_file.$$"
```

This way the user can override the `/tmp`, and adding the PID makes it less likely to conflict with an existing file.

(It seems that mktemp(1) is not in POSIX. Oh well.)